### DISCUSSION TITLE
Difference Between SNOW and SNOW2

### DISCUSSION THREAD

**Discussion by JahanzebTariq** [PMEAL/OpenPNM#2655] [Q&A]
Greetings @ma-sadeghi and @jgostick .. I see that there has been **a lot of improvements** going on with the **Network Extraction Algorithm**, could you please tell me what are the **differences between SNOW and SNOW2**. **Is it just the accuracy** or something else too. I see we have **to do somethings manually too in SNOW2** which were **automated in SNOW**, But I would like to here from you the **comparison between the two algorithms**. 
Thanks in advance

**Comment by jgostick** [comment]
Snow2 has several new features and a different set of arguments to the function, such as how borders are handled.  It has the ability to run parallelized and to do multiphase.  We also changed some of the underlying functions to run more quickly and be more robust.  This included changing how peaks were found, with the result that snow2 will return a network with more pores.  It is possible to pass an image containing peaks into snow2 so that you can get the old behavior.
